contracts/hardhat.config.zksync.js

```
@@ -22,11 +22,13 @@ module.exports = {
     ],
   },
   zksolc: {
-    version: "1.3.16", // Latest versions aren't supported for verification yet.
-    settings: {},
+    version: "1.5.11",
+    settings: {
+      suppressedErrors: ["sendtransfer"]
```

ESLint

[error] Insert `,`

(prettier/prettier)